### PR TITLE
Migration bugfix

### PR DESCRIPTION
**Pull request by calebpower**
- fixed migrations in meeting
- rolled back two migrations, added one
- changed default date to UNIX epoch

### CODE REVIEW

File: meeting/migrations/0009_auto_20200311_2209.py [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
+            model_name='meeting',
+            name='end_date',
+            field=models.DateField(default='1970-01-01'),
+        ),
+        migrations.AddField(
+            model_name='meeting',
+            name='start_date',
+            field=models.DateField(default='1970-01-01'),
+        ),
+    ]
```
Comment (calebpower) [inline meeting/migrations/0009_auto_20200311_2209.py:27]: had to rewrite this file to compensate for the new changes

File: meeting/models.py [hunk 2]
```diff
@@ -45,8 +45,8 @@ def __str__(self):
         return "meeting " + str(self.meeting) + " from " + str(self.start_time) + " to " + str(self.end_time)
 
 class Meeting(models.Model):
-    start_date = models.DateField(default = '0000/00/00')
-    end_date = models.DateField(default = '0000/00/00')
+    start_date = models.DateField(default = '1970-01-01')
+    end_date = models.DateField(default = '1970-01-01')
```
Comment (calebpower) [inline meeting/models.py:49]: Migrations didn't like the YYYY/MM/DD format (didn't like the slashes) and it didn't like a zero-date, so I set it to the UNIX epoch.

**Comment by calebpower** [comment]
@capple2016 Preemptively merging this one into dev so that other's merges don't break. We can revert this later if need be.

File: meeting/models.py (hunk 2 above)
Comment (capple2016) [reply meeting/models.py:49]: That'll work. Thank you for fixing it.

Comment (calebpower) [reply meeting/models.py:49]: absolutely.

**Pull request merged**